cmd/config.go

```diff
@@ -325,10 +325,11 @@ type GRPCClientConfig struct {
 
 // GRPCServerConfig contains the information needed to run a gRPC service
 type GRPCServerConfig struct {
-	Address               string `json:"address" yaml:"address"`
-	ServerCertificatePath string `json:"serverCertificatePath" yaml:"server-certificate-path"`
-	ServerKeyPath         string `json:"serverKeyPath" yaml:"server-key-path"`
-	ClientIssuerPath      string `json:"clientIssuerPath" yaml:"client-issuer-path"`
+	Address               string   `json:"address" yaml:"address"`
+	ServerCertificatePath string   `json:"serverCertificatePath" yaml:"server-certificate-path"`
+	ServerKeyPath         string   `json:"serverKeyPath" yaml:"server-key-path"`
+	ClientIssuerPath      string   `json:"clientIssuerPath" yaml:"client-issuer-path"`
+	ClientWhitelist       []string `json:"clientWhitelist" yaml:"client-whitelist"`
```

Comment:
In the interests of decolonizing language, let's just call this `ClientNames`. Let's also document the fact that they can be DNS names or IP addresses.